src/engine/controls/bpmcontrol.cpp

```diff
@@ -183,42 +188,49 @@ void BpmControl::slotFileBpmChanged(double file_bpm) {
 }
 
 void BpmControl::slotAdjustBeatsFaster(double v) {
-    BeatsPointer pBeats = m_pBeats;
-    if (v > 0 && pBeats && (pBeats->getCapabilities() & Beats::BEATSCAP_SETBPM)) {
+    mixxx::BeatsPointer pBeats = m_pBeats;
+    if (v > 0 && pBeats && (pBeats->getCapabilities() & mixxx::Beats::BEATSCAP_SETBPM)) {
         double bpm = pBeats->getBpm();
         double adjustedBpm = bpm + kBpmAdjustStep;
         pBeats->setBpm(adjustedBpm);
     }
 }
 
 void BpmControl::slotAdjustBeatsSlower(double v) {
-    BeatsPointer pBeats = m_pBeats;
-    if (v > 0 && pBeats && (pBeats->getCapabilities() & Beats::BEATSCAP_SETBPM)) {
+    mixxx::BeatsPointer pBeats = m_pBeats;
```

Comment:
Assigning the member variable to a local variable doesn't add any value. It just introduces noise.

Reply:
Done that one and some other found in the same file. Just a comment on that.
The code was already like that, I just added the namespace in a semi automated way. In general I have tried to reduce the modifications of code that was not in the scope of my work to try to simplify the review of the PR. Is it expected that we do this kind of fixes mixed in the PR's?

Reply:
Yes, we improve code as we work on it. Otherwise we would have bad code around forever. We do not do mass find-and-replace changes because it causes lots of merge conflicts with open PRs.